fun will_throw
{
try { fun will_throw
{
void
try {
! 2[ will_throw()]
}
catch const e {
}return 3
}
var a   = void
var exc =[ will_throw()]
}
catch const e {
}return 13
}
var exc = void
try {[ will_throw()... ]
}
catch const e {
 e
}
assert exc
